### ISSUE TITLE
[autonity-network] Support exported metrics for the new Autonity

### ISSUE THREAD

**Issue by yazzaoui** [clearmatics/charts-ose#62]
As a user/operator/developer
I want to have access to autonity internal metrics with the 1.9.1 update 
So that we can monitor the network
It would be nice to have as well as preconfigured the official go-ethereum grafana dashboard

### Acceptance criteria
- Autonity `dev-0.1.6-ethstats` correctly exports metrics data

### Ressources
- https://blog.ethereum.org/2019/07/10/geth-v1-9-0/#metrics-collection
- [json dashboard](https://gist.githubusercontent.com/karalabe/e7ca79abdec54755ceae09c08bd090cd/raw/3a400ab90f9402f2233280afd086cb9d6aac2111/dashboard.json)

### TODO
- [x] Up docker images:
    - `autonity` to `v0.1.6` 
    - `autonity-init` to `v0.1.1`
- [x] Change autonity args to expose prometheus and disable old metrics
- [x] disable geth prometheus exporter sidecar 
- [x] change service `validator-$i` for new ports
- [x] change prometheus autodiscovery rules if it needs
- [x] change grafana dashboard data source and template 

### Blocked by
- [x] issues #66
